integrations/GA4/ECommerceEventConfig.js

```diff
@@ -129,14 +132,22 @@ const eventNamesConfigArray = [
     ],
     hasItem: true,
   },
-  {
-    src: ["payment info entered"], // adding item_name as checkout_id. I know its not feasible but what is correct value to send.
-    dest: "add_payment_info",
-    requiredParams: [
-      requiredEventParameters.ProductId,
-      requiredEventParameters.ProductName,
+  // To handle sending multiple payload for single event use approach as below
+  {
+    src: ["payment info entered"],
+    dest: [
+      {
+        dest: "add_payment_info",
+        hasItem: false,
+        onlyIncludeParams: includeParams.PaymentInfo,
+      },
+      {
+        dest: "add_shipping_info",
+        hasItem: false,
+        onlyIncludeParams: includeParams.ShippingInfo,
+      },
     ],
-    hasItem: true,
+    hasMultiplePayload: true,
```

Comment:
do we need this?
{
    src: ["payment info entered"],
    dest:  "add_payment_info",
    hasItem: false,
    onlyIncludeParams: includeParams.PaymentInfo,
 },
 
 {
  src: ["payment info entered"],
 dest: "add_shipping_info",
 hasItem: false,
 onlyIncludeParams: includeParams.ShippingInfo,
 },
 
 and filter multiple events by same src name?

Reply:
we can do this as well